fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -1322,13 +1377,12 @@ ACTOR static Future<Void> transactionStarter(
 			auto& req = transactionQueue.top().first;
 			int tc = req.transactionCount;
 
-			if (req.priority() < GetReadVersionRequest::PRIORITY_DEFAULT &&
-			    !batchRateInfo.canStart(transactionsStarted[0] + transactionsStarted[1])) {
-				break;
-			} else if (req.priority() < GetReadVersionRequest::PRIORITY_SYSTEM_IMMEDIATE &&
-			           !normalRateInfo.canStart(transactionsStarted[0] + transactionsStarted[1])) {
+			if(req.priority() < GetReadVersionRequest::PRIORITY_DEFAULT && !batchRateInfo.canStart(transactionsStarted[0] + transactionsStarted[1], tc)) {
```

Comment:
It seems this PR has not applied clang-format. 
I also saw some trailing space after checking it out locally. 
Would you mind clang-formatting  the change?